backend/src/db/pipelines/compliance/softwareDetailAggregation.ts

```diff
@@ -16,7 +16,7 @@ export const softwareDetailAggregationPipeline = (softwareName: string): any[] =
             "bbVersion": "$$detail.v.bbVersion",
             "requirements": "$$detail.v.requirementSpecificationCompliance",
             "interface": "$$detail.v.interfaceCompliance",
-            "deploymentCompliance": "$deploymentCompliance.requirements" 
+            "deploymentCompliance": "$$detail.v.deploymentCompliance" 
```

Comment:
is second dolar sign intentional?

Reply:
No, it is a typo. It work in both with single and double but the single one is a proper syntax